include/mgp.hpp

```diff
@@ -2723,6 +2838,43 @@ inline bool Path::operator==(const Path &other) const { return util::PathsEqual(
 
 inline bool Path::operator!=(const Path &other) const { return !(*this == other); }
 
+inline const std::string Path::ToString() const {
+  const auto length = Length();
+  size_t i = 0;
+  std::string return_string = "";
+  for (i = 0; i < length; i++) {
+    std::string labels{""};
+    const auto node = GetNodeAt(i);
+    for (auto label : node.Labels()) {
+      labels += ":" + std::string(label) + " ";
+    }
+    return_string += "(id: " + std::to_string(node.Id().AsInt()) + " " + labels + ")-";
+
+    const Relationship rel = GetRelationshipAt(i);
+    std::string properties = "";
+    std::map<std::string, Value> properties_map{rel.Properties()};
+    const size_t map_size = properties_map.size();
+    size_t i = 0;
+    for (auto item : properties_map) {
+      if (i == map_size - 1) {
+        properties += std::string(item.first) + "=" + item.second.ToString();
+        break;
+      }
+      properties += std::string(item.first) + "=" + item.second.ToString() + ", ";
+      ++i;
+    }
+    return_string += "[type: " + std::string(rel.Type()) + ", id: " + std::to_string(rel.Id().AsInt()) +
+                     ", properties: {" + properties + "}]->";
+  }
```

Comment:
Here then you can use `relationship.ToString()` if I am not wrong

Reply:
I dont think so, because relationship to string also outputs nodes included in the relationship, so the nodes would be duplicated in the string